modules/appeals_api/spec/fixtures/v2/valid_200996_extra.json

```diff
@@ -1,189 +1,189 @@
 {
-    "data": {
-      "type": "higherLevelReview",
-      "attributes": {
-        "benefitType": "education",
-        "veteran": {
-          "homeless": true,
-          "address": {
-            "addressLine1": "123 Main St",
-            "addressLine2": "Suite #1200",
-            "addressLine3": "Box 4",
-            "city": "New York",
-            "countryCodeISO2": "US",
-            "stateCode": "NY",
-            "zipCode5": "30012"
-          },
-          "phone": {
-            "countryCode": "34",
-            "areaCode": "555",
-            "phoneNumber": "8001111",
-            "phoneNumberExt": "2"
-          },
-          "email": "123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789_123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789_12345678901234567890@bobbytablesemail.com",
-          "timezone": "America/Chicago"
+  "data": {
```

Comment:
Since you've made changes to this file, you'll almost certainly need to regenerate the `expected_200996_extra.pdf` to get the existing extra test to pass as well :)